springboot-mvc/src/main/resources/application.properties

```diff
@@ -0,0 +1,13 @@
+spring.h2.console.enabled=true
+spring.datasource.platform=h2
+spring.datasource.url=jdbc:h2:mem:contact
```

Comment:
You must need to use a real open source database

Reply:
Sorry got the task wrong, I thought this was meant to stay as h2, and second task use open source DB